test/cpp/end2end/client_callback_end2end_test.cc

```diff
@@ -517,13 +524,19 @@ TEST_P(ClientCallbackEnd2endTest, SequentialGenericRpcs) {
 TEST_P(ClientCallbackEnd2endTest, SequentialGenericRpcsAsBidi) {
   MAYBE_SKIP_TEST;
   ResetStub();
-  SendGenericEchoAsBidi(10, 1);
+  SendGenericEchoAsBidi(10, 1, /*do_writes_done=*/true);
```

Comment:
Is it useful to do combos of writesdone and not for all of these tests, either through parameterized testing or by just sending two RPCs each time?

Reply:
Sounds good, will do so in a followup PR. In this one, I just wanted a repro for the failure. I think it's also worth testing both destruction orders (channel first, server first).